bottles/frontend/views/bottle_preferences.py

```diff
@@ -194,11 +194,15 @@ def __init__(self, details, config, **kwargs):
         self.switch_vmtouch.set_sensitive(vmtouch_available)
         _not_available = _("This feature is unavailable on your system.")
         _flatpak_not_available = _("{} To add this feature, please run flatpak install").format(_not_available)
+        _gamescope_pkg_name = "org.freedesktop.Platform.VulkanLayer.gamescope"
+        _vkbasalt_pkg_name = "org.freedesktop.Platform.VulkanLayer.vkBasalt"
+        _mangohud_pkg_name = "org.freedesktop.Platform.VulkanLayer.MangoHud"
+        _obsvkc_pkg_name = "com.obsproject.Studio.Plugin.OBSVkCapture"
         _flatpak_pkg_name = {
-            "gamescope": "org.freedesktop.Platform.VulkanLayer.gamescope",
-            "vkbasalt": "org.freedesktop.Platform.VulkanLayer.vkBasalt//22.08",
-            "mangohud": "org.freedesktop.Platform.VulkanLayer.MangoHud//22.08",
-            "obsvkc": "com.obsproject.Studio.Plugin.OBSVkCapture"
+            "gamescope": f"{_gamescope_pkg_name}//{base_version}" if base_version else _gamescope_pkg_name,
+            "vkbasalt": f"{_vkbasalt_pkg_name}//{base_version}" if base_version else _vkbasalt_pkg_name,
+            "mangohud": f"{_mangohud_pkg_name}//{base_version}" if base_version else _mangohud_pkg_name,
+            "obsvkc": _obsvkc_pkg_name
```

Comment:
Same deal, no need to check `base_version`. Just the `{*_pkg_name}//{base_version}"` bit works. No need to inline the names here in future changes, since that'll just be a bit more pain.